super_resolution/batchsuperres.py

```diff
@@ -0,0 +1,51 @@
+import subprocess, os, fnmatch, time, datetime, argparse

```

Comment:
the format you're using is very different from any of our other examples, I'd suggest you take a look at them and see how your code can mimic it structurally